atlp-rwanda/rca-barefoot-team6-fe
# Ft forgot password

#### What does this PR do?
 Implementing forgot password on the frontend.
 ... 

#### Description of  the task to be completed?
I have done the implementation of forgot password where verified user can enter his email in the field and receive an email link
...

#### How should this be manually tested?
Check out this branch and then run npm i after successful installation run npm run dev then go into browser at this URL {BASE_URL}/auth/forgot-password
...

#### Any background context you want to provide?
Create configuration file in plugins folder that export backendUrl  to be used & add it to gitignore to hide url
...

#### What are the relevant pivotal tracker stories?
#184986709
...

#### Screenshots (if appropriate)

...
